.jshintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "node": true,
+
+  "curly": true,
+  "latedef": true,
+  "quotmark": true,
+  "undef": true,
+  "unused": true,
+  "trailing": true
+}
```

Comment:
is this file meant to be here? style config currently lives in https://github.com/skatejs/named-slots/blob/master/.eslintrc